ansible/group_vars/all.yml

```
@@ -17,7 +17,8 @@ runtime_socket: /var/run/docker.sock
 
 excluded_pairs:
   # e.g. - ['ubuntu-1804-lts', 'core_bpf']
-  -
+  - ['fedora-coreos', 'core_bpf'] # Failing due to verifier errors
+  - ['fedora-coreos', 'ebpf'] # No drivers for kernel 6.3+, see ROX-16705
```

The test is still spawned; could it be that the `family` is expected here ?
```suggestion
  - ['fedora-coreos-stable', 'core_bpf'] # Failing due to verifier errors
  - ['fedora-coreos-stable', 'ebpf'] # No drivers for kernel 6.3+, see ROX-16705
```

---

I thought the family was `fedora-coreos`, no harm in trying though.